src/com/lilithsthrone/game/character/PlayerCharacter.java

```diff
@@ -1649,6 +1651,14 @@ public boolean isDiscoveredWorldMap() {
 		return this.isQuestProgressGreaterThan(QuestLine.MAIN, Quest.MAIN_2_D_MEETING_A_LILIN);
 	}
 
+	public boolean isInActive() {
+		return !isActive;
+	}
+
+	public void setActive(boolean active) {
+		isActive = active;
+	}
```

Comment:
Can't `setActive()` have a default `True` boolean, as the name suggest?

Can't people use `player.isActive = False` instead?

Reply:
That just entered my mind while I was doing something else, so yes, it needs a default value true. 

isActive has protected access, so you can only access in this class, or with this public method. It's fairly common to do it this way.